post: configure a  modem in Bridge mode for UDM (#108)
cloudx-labs/posts: posts/navarroaxel/15.modem-in-bridge-mode-for-udm.md

---
title: 'Setting Up a Modem in Bridge Mode for a UDM router: A Step-by-Step Guide'
published: true

> description: Learn how to configure your modem in bridge mode to use it with a Ubiquiti Dream Machine, allowing the UDM to handle the PPPoE connection

Reviewer: ```suggestion
description: Learn how to configure your modem in bridge mode to use it with a Unifi Dream Machine, allowing the UDM to handle the PPPoE connection
```

tags: 'networking, network, unifi, modem'
cover_image: ./assets/networking-bridge-mode.png
id: 1596373
---

> I bought an Ubiquiti Dream Machine (UDM) and then I called my ISP to change the modem to bridge mode, but they don't provide support for that. You should do it by yourself.

Reviewer: ```suggestion
I bought a Unifi Dream Machine (UDM) and then I called my ISP to change the modem to bridge mode, but they don't provide support for that. You should do it by yourself.
```
